docs/integrations/sources/surveymonkey.md

```diff
@@ -1,53 +1,53 @@
-# SurveyMonkey
+# Set up the SurveyMonkey Source Connector
```

Comment:
```suggestion
# SurveyMonkey

This page guides you through setting up the SurveyMonkey source connector.
```
```suggestion
# Set up the SurveyMonkey Source Connector
```

Reply:
This is where the navigation bar gets the name of the source to include in the source catalog. As is, when you look for this source in the sources list, it's listed as "Set up the SurveyMonkey Source Connector".

Reply:
![Screenshot 2024-06-24 at 11 56 32 AM](https://github.com/airbytehq/airbyte/assets/23410478/204e5a69-cd83-413c-9666-614a38794e04)